okhttp/src/main/java/okhttp3/internal/http1/Http1Codec.java

```diff
@@ -18,6 +18,8 @@
 import java.io.EOFException;
 import java.io.IOException;
 import java.net.ProtocolException;
+import java.net.SocketTimeoutException;
+
```

Comment:
No spaces between imports. I think IntelliJ adds them by default, you'll want to change that.